### PR TITLE
docs: update mono-package documentation

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

- Updated documentation related to mono-package
- Created a blog post so we can share it on multiple channels and let people know about dialtone 9.

@francisrupert @yorbi-dp @fede-dp please help me verifying that the redaction is correct before publishing this.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [ ] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md
Comment (yorbi-dp) [inline apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md:56]: This does not direct me to the 'Import Packages' section.

**Review by iropolo — APPROVED** [review]
Approve, lets wait for design review. Thanks,

File: README.md
Comment (braddialpad) [inline README.md:117]: If this is the primary way to release, should we just make it `pnpm run release`?

File: README.md
Comment (braddialpad) [inline README.md]: Call this "usage" to avoid confusion whether this is installation for development or usage of the library.

File: README.md
Comment (braddialpad) [inline README.md]: lets just label this CSS instead of LESS, since @import is not specific to LESS.

File: README.md
Comment (braddialpad) [inline README.md]: Let's also state that you should avoid importing icons directly and use the dialtone vue icon component.

File: README.md
Comment (braddialpad) [inline README.md]: Maybe lets do an example component instead of * since we never really want anyone importing *, and people tend to copy paste stuff directly from the docs.

File: apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md
Comment (braddialpad) [inline apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md]: Maybe instead of putting a direct command, just say "If you have any of these packages installed, they will need to be uninstalled". Since we may not know what exactly is installed in different projects.

- @dialpad/dialtone-icons
- @dialpad/dialtone-tokens
- @dialpad/dialtone-vue
- @dialpad/dialtone
- @dialpad/eslint-plugin-dialtone

File: apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md
Comment (braddialpad) [inline apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md]: Hmm maybe just remove this. We'll probably do the migration on ubervoice but not any other projects. I can just include this in the message linking to this post when I send it out.

File: packages/dialtone-css/README.md
Comment (braddialpad) [inline packages/dialtone-css/README.md]: Can probably just remove this entire thing it is talking too much about Dialtone in general. Say something like

"This is the library for Dialtone's CSS. Any classes or styles used within Dialtone should be stored here and documented on our site in apps/dialtone-documentation."

File: apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md
Comment (braddialpad) [inline apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md]: "increased the major version"

**Review by braddialpad — COMMENTED** [review]
Happy new year! Looks pretty good, a couple small things.

File: README.md
Comment (juliodialpad) [reply README.md:117]: Changed to be more explicit about what this does, now it just only release the affected projects.

If we want to release every project we need to run `pnpm run -r release` but it's not necessary trying to release every project every time we want to release, this improves the releasing speed process a bit.

File: apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md
Comment (juliodialpad) [reply apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md]: Makes sense, removing

File: apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md
Comment (juliodialpad) [reply apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md:56]: Once merged, it will work correctly.

You can try it changing the branch on GitHub, I think it will not work directly on the diff.

https://github.com/dialpad/dialtone/blob/docs/update-monopackage-documentation/apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md

**Comment by juliodialpad** [comment]
> Happy new year! Looks pretty good, a couple small things.

Changes are in, Happy new year! 🥳

File: README.md
Comment (braddialpad) [reply README.md:117]: k sounds good.

**Review by braddialpad — APPROVED** [review]
Great thanks!

**Pull request merged**
